test/indexer/test_hca_indexer.py

```diff
@@ -442,77 +442,78 @@ def _num_docs_by_index(self, hits) -> Mapping[Tuple[str, bool], int]:
             counter[entity_type, aggregate] += 1
         return counter
 
-    def test_contributor_matrices(self):
+    def test_indexed_matrices(self):
         """
-        Test indexing of multiple contributor-generated matrix bundles including
-        analysis file and supplementary file CGM.
+        Test indexing various types of DCP and contributor generated matrix
+        bundles including analysis file and supplementary file CGM.
```

Comment:
Something's definitely ungrammatical about these line but I'm not certain what the correct form would be. Here's a suggestion that may not be entirely accurate: 

"Test indexing various types of DCP- and contributor-generated matrix bundles, including analysis and supplementary file CGMs."